developerportal/templates/header.html

```diff
@@ -64,6 +64,13 @@ <h4 class="mzp-c-menu-item-title">{{ page.title }}</h4>
                       </li>
                     {% endwith %}
                   {% endfor %}
+                  {% if BLOG_LINK %}
+                  <li class="mzp-c-menu-category">
+                      <a class="mzp-c-menu-title" href="{{ BLOG_LINK }}">
```

Comment:
```suggestion
                      <a class="mzp-c-menu-title" href="{{ BLOG_LINK }}" rel="noopener">
```

Reply:
I've separately removed the `noopener`s in general when `target` wasn't `_blank`